lib/ansible/utils/module_docs_fragments/azure.py

```diff
@@ -67,19 +67,27 @@ class ModuleDocFragment(object):
               C(AzureUSGovernment)), or a metadata discovery endpoint URL (required for Azure Stack). Can also be set via credential file profile or
               the C(AZURE_CLOUD_ENVIRONMENT) environment variable.
         default: AzureCloud
+    cli_default_profile:
+        description:
+            - Set to C(true), when login with Azure CLI
+        required: false
```

Comment:
If this is a new option then please add
`version_added: "2.5"`

Reply:
It is a bugfix, the code has this option already, but the document haven't mention this option before.

Reply:
ah, OK. Thanks for clarification